modules/social_features/social_group/modules/social_group_secret/social_group_secret.install

```diff
@@ -0,0 +1,58 @@
+<?php
+
+/**
+ * @file
+ * Install, update and uninstall functions for the social_group_secret module.
+ */
+
+/**
+ * Implements hook_install().
+ */
+function social_group_secret_install() {
+  // Set some default permissions.
+  _social_group_secret_set_permissions();
+
+  // Set module weight.
```

Comment:
Can we replace this comment by a description on _why_ we do this? The next line already tells us what we're doing but knowing why we do this can save someone a few hours in a few months time :)

Reply:
This still needs work